<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="17.2070" tests="7" failures="2">
  <testsuite name="Root Suite" timestamp="2022-09-01T19:32:08" tests="0" file="cypress\e2e\PfizerWeb\inscripcion.cy.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Test Sección Inscripción" timestamp="2022-09-01T19:32:08" tests="7" time="17.2070" failures="2">
    <testcase name="Test Sección Inscripción Inscripción Sin Producto" time="5.3990" classname="Inscripción Sin Producto">
    </testcase>
    <testcase name="Test Sección Inscripción Inscripción Sin Médico" time="2.8060" classname="Inscripción Sin Médico">
    </testcase>
    <testcase name="Test Sección Inscripción Inscripción Con Médico Otro Sin Completarlo" time="3.4430" classname="Inscripción Con Médico Otro Sin Completarlo">
    </testcase>
    <testcase name="Test Sección Inscripción Inscripción Con Producto que No Esta en el País" time="2.4200" classname="Inscripción Con Producto que No Esta en el País">
    </testcase>
    <testcase name="Test Sección Inscripción Inscripción Con un Médico que No Esta en el País" time="3.1390" classname="Inscripción Con un Médico que No Esta en el País">
    </testcase>
    <testcase name="Test Sección Inscripción Inscripción Seleccionando Datos de un País y luego Cambiandolo" time="0.0000" classname="Inscripción Seleccionando Datos de un País y luego Cambiandolo">
      <failure message="Timed out retrying after 4050ms: `cy.select()` failed because this element:

`&lt;select class=&quot;form-control form-control-pl &quot; id=&quot;select_pais&quot;&gt;...&lt;/select&gt;`

is being covered by another element:

`&lt;div class=&quot;jconfirm-cell&quot;&gt;...&lt;/div&gt;`

Fix this problem, or use {force: true} to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with" type="CypressError"><![CDATA[CypressError: Timed out retrying after 4050ms: `cy.select()` failed because this element:

`<select class="form-control form-control-pl " id="select_pais">...</select>`

is being covered by another element:

`<div class="jconfirm-cell">...</div>`

Fix this problem, or use {force: true} to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with
    at $Cy.ensureDescendents (http://192.168.0.30/__cypress/runner/cypress_runner.js:162000:78)
    at ensureDescendents (http://192.168.0.30/__cypress/runner/cypress_runner.js:147392:8)
    at ensureDescendentsAndScroll (http://192.168.0.30/__cypress/runner/cypress_runner.js:147399:14)
    at ensureElIsNotCovered (http://192.168.0.30/__cypress/runner/cypress_runner.js:147531:5)
    at runAllChecks (http://192.168.0.30/__cypress/runner/cypress_runner.js:147721:52)
    at retryActionability (http://192.168.0.30/__cypress/runner/cypress_runner.js:147745:16)
    at tryCatcher (http://192.168.0.30/__cypress/runner/cypress_runner.js:13012:23)
    at Function.Promise.attempt.Promise.try (http://192.168.0.30/__cypress/runner/cypress_runner.js:10286:29)
    at whenStable (http://192.168.0.30/__cypress/runner/cypress_runner.js:166930:65)
    at http://192.168.0.30/__cypress/runner/cypress_runner.js:166385:14
    at tryCatcher (http://192.168.0.30/__cypress/runner/cypress_runner.js:13012:23)
    at Promise._settlePromiseFromHandler (http://192.168.0.30/__cypress/runner/cypress_runner.js:10947:31)
    at Promise._settlePromise (http://192.168.0.30/__cypress/runner/cypress_runner.js:11004:18)
    at Promise._settlePromise0 (http://192.168.0.30/__cypress/runner/cypress_runner.js:11049:10)
    at Promise._settlePromises (http://192.168.0.30/__cypress/runner/cypress_runner.js:11129:18)
    at Promise._fulfill (http://192.168.0.30/__cypress/runner/cypress_runner.js:11073:18)
    at http://192.168.0.30/__cypress/runner/cypress_runner.js:12687:46
From Your Spec Code:
    at Context.eval (http://192.168.0.30/__cypress/tests?p=cypress\e2e\PfizerWeb\inscripcion.cy.js:208:33)]]></failure>
    </testcase>
    <testcase name="Test Sección Inscripción Inscripción Agregando mas de una vez el Mismo Producto y Médico" time="0.0000" classname="Inscripción Agregando mas de una vez el Mismo Producto y Médico">
      <failure message="Timed out retrying after 4050ms: `cy.click()` failed because the center of this element is hidden from view:

`&lt;a href=&quot;/ViatrisWeb/InscripcionPacientes&quot;&gt;Inscrip...&lt;/a&gt;`

Fix this problem, or use `{force: true}` to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with" type="CypressError"><![CDATA[CypressError: Timed out retrying after 4050ms: `cy.click()` failed because the center of this element is hidden from view:

`<a href="/ViatrisWeb/InscripcionPacientes">Inscrip...</a>`

Fix this problem, or use `{force: true}` to disable error checking.

https://on.cypress.io/element-cannot-be-interacted-with
    at $Cy.ensureDescendents (http://192.168.0.30/__cypress/runner/cypress_runner.js:162016:76)
    at ensureDescendents (http://192.168.0.30/__cypress/runner/cypress_runner.js:147392:8)
    at possiblyScrollMultipleTimes (http://192.168.0.30/__cypress/runner/cypress_runner.js:147499:22)
    at scrollContainers (http://192.168.0.30/__cypress/runner/cypress_runner.js:147522:16)
    at ensureDescendentsAndScroll (http://192.168.0.30/__cypress/runner/cypress_runner.js:147526:14)
    at ensureElIsNotCovered (http://192.168.0.30/__cypress/runner/cypress_runner.js:147531:5)
    at runAllChecks (http://192.168.0.30/__cypress/runner/cypress_runner.js:147721:52)
    at retryActionability (http://192.168.0.30/__cypress/runner/cypress_runner.js:147745:16)
    at tryCatcher (http://192.168.0.30/__cypress/runner/cypress_runner.js:13012:23)
    at Function.Promise.attempt.Promise.try (http://192.168.0.30/__cypress/runner/cypress_runner.js:10286:29)
    at whenStable (http://192.168.0.30/__cypress/runner/cypress_runner.js:166930:65)
    at http://192.168.0.30/__cypress/runner/cypress_runner.js:166385:14
    at tryCatcher (http://192.168.0.30/__cypress/runner/cypress_runner.js:13012:23)
    at Promise._settlePromiseFromHandler (http://192.168.0.30/__cypress/runner/cypress_runner.js:10947:31)
    at Promise._settlePromise (http://192.168.0.30/__cypress/runner/cypress_runner.js:11004:18)
    at Promise._settlePromise0 (http://192.168.0.30/__cypress/runner/cypress_runner.js:11049:10)
    at Promise._settlePromises (http://192.168.0.30/__cypress/runner/cypress_runner.js:11129:18)
    at Promise._fulfill (http://192.168.0.30/__cypress/runner/cypress_runner.js:11073:18)
    at http://192.168.0.30/__cypress/runner/cypress_runner.js:12687:46
From Your Spec Code:
    at Context.eval (http://192.168.0.30/__cypress/tests?p=cypress\e2e\PfizerWeb\inscripcion.cy.js:216:45)]]></failure>
    </testcase>
  </testsuite>
</testsuites>